torch/csrc/jit/runtime/static/memory_planner.cpp

```diff
@@ -135,6 +135,44 @@ std::vector<StorageGroup> assignStorageToManagedTensors(
   return managed_tensor_groups;
 }
 
+ManagedStorages::ManagedStorages()
+    : storages_(nullptr), size_(0), capacity_(0) {}
+
+ManagedStorages::~ManagedStorages() {
+  deallocate();
+}
+
+void ManagedStorages::allocate(size_t capacity) {
+  if (storages_ != nullptr) {
+    deallocate();
+  }
+  capacity_ = capacity;
+  storages_ = reinterpret_cast<at::StorageImpl*>(
+      new unsigned char[capacity_ * sizeof(at::StorageImpl)]);
```

Comment:
You gonna set `size_`?

Reply:
Setting `size_ = 0` here shouldn't be necessary. If `storages_ == nullptr`, then `size_` is already 0. Otherwise, `deallocate()` will set it to 0

Reply:
But it would probably be a good idea to add an internal assert that `size_ == 0` just to make sure